IntAllocatorBase.cpp

```diff
@@ -0,0 +1,46 @@
+class IntAllocatorBase {
+	public:
+		virtual int* allocate(size_t size) = 0;
```

Comment:
Where is the implementation for this pure virtual method?

Reply:
I understood from the notice that i can use it as an implemented func...
I thought that i have to implement the add/remove func and use the de/allocat func